### ISSUE TITLE
Reproduce Numbers on CIFAR100

### ISSUE THREAD

**Issue by YUE-FAN** [TorchSSL/TorchSSL#20]
Hi,

Thanks for the great work. I tried to reproduce FlexMatch number on CIFAR100 with 400 labels. I followed [the instructions](https://github.com/TorchSSL/TorchSSL/blob/main/README.md) to create a conda env and ran `python flexmatch.py --c config/flexmatch/flexmatch_cifar100_400_0.yaml` with 3 Tesla V100.

Unfortunately, the best top-1 accuracy I got was 60.65, which is 6.91% lower than the reported number in the paper. There seems to be a sharp performance drop at around iteration 600K. But I couldn't pinpoint the issue from the training statistics. I wonder if you also have observed similar behaviors. It would be great if you could offer some insights here. Thanks!
![image](https://user-images.githubusercontent.com/35504790/145006318-e623c79b-7dc6-42c4-804c-450406bc2f04.png)

![image](https://user-images.githubusercontent.com/35504790/145006501-5d6e917a-766d-4b29-814a-b0afafa5f156.png)

Here is the tensorboard file: [tf_logs.zip](https://github.com/TorchSSL/TorchSSL/files/7667273/tf_logs.zip)

Besides, the curve of the mask ratio looks a bit strange to me.  Because `1.0 - mask.detach()` is actually logged in the code, so, shouldn't it start from 1 and then decrease? Any intuition why it starts from 0 and increases at the very beginning? Thanks!

**Comment by Linkin-9527** [comment]
Dear authors,
Thanks for the impressive work FlexMatch !
I also have this problem. The accuracy of CIFAR100  with 400 labels is very low.
Thanks for your time!

**Comment by qianlanwyd** [comment]
Hi, all
Thank you for the interest of our work.
Previously we used the internal codes to run all the experiments. And the environment will also affect the results.  I have been using hundreds of GPUs of Azure to re-run all the experiments in the paper. I will clarify the gpu for each algorithm and dataset. Besides I will make all the log files available in TorchSSL git repo. Please wait for my updates. I will gradually upload all log files in 2 moths.

**Comment by qianlanwyd** [comment]
Hi, we updated the results. Please check.

**Comment by ZiTingW** [comment]
Hi, 
I observed very similar top-1 acc curves, with a sharp drop in performance at some iterations (like the 600k mentioned above). I wonder if the results reported in the paper are the best accuracy over the entire training process or the best acc in the last few checkpoints.

**Comment by qianlanwyd** [comment]
the best accuracy over the entire training process

ZiTingW ***@***.***>于2022年1月25日 周二15:14写道：

**Comment by ZiTingW** [comment]
ok, thank you.
